vgstation13.dme

```diff
@@ -2832,7 +2832,7 @@
 #include "maprendering\maprendering.dm"
 #include "maps\_map.dm"
 #include "maps\_map_override.dm"
-#include "maps\tgstation.dm"
+#include "maps\test_tiny.dm"
```

Comment:
💣

Reply:
now that I'm wrapping up I really need to remember to remove it though